docs/CodeDevelopment/requirements.md

```diff
@@ -64,6 +99,13 @@
 2. Use STATIC on each non-public function and non-public global to avoid conflicts with other modules.
 3. Use EFIAPI on all public library class functions.
 
+## Source files
+
+Source files should follow the Edk2 coding standards available on [TianoCore.org](https://www.tianocore.org/) website.  
```

Comment:
I think it would be helpful to directly link to the C Coding Standards Specification.

https://edk2-docs.gitbook.io/edk-ii-c-coding-standards-specification/

Reply:
Will update link and text to point directly to coding standard.